automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/WebBrowserSpec.scala

```diff
@@ -54,6 +54,7 @@ trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogg
     val service = new ChromeDriverService.Builder().usingDriverExecutable(new File(Config.ChromeSettings.chromDriverPath)).usingAnyFreePort().build()
     service.start()
     implicit val driver = new RemoteWebDriver(service.getUrl, getChromeIncognitoOption)
+    driver.manage.window.setSize(new org.openqa.selenium.Dimension(1600, 2400))
```

Comment:
I know I'm late to the game here but does this work if the screen resolution is less than the requested dimensions?